src/Common/src/Interop/Unix/Interop.Libraries.cs

```diff
@@ -6,6 +6,10 @@ internal static partial class Interop
 {
     internal static partial class Libraries
     {
+#if MONO
```

Comment:
You should just create a new partial class Libraries file with MONO settings and include that in build